data.sql

```diff
@@ -5,3 +5,28 @@ INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts)
 INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Pikachu', '2021-01-07', 15.04, false, 1);
```

Comment:
Please add a custom project title and project descriptions in the README file. This way it looks more professional.